python-avd/tests/pyavd/utils/password/test_password.py

```diff
@@ -5,14 +5,9 @@
 
 __metaclass__ = type
```

Comment:
Please remove __metaclass__ and repalce from __future__ with `from __future__ import annotations`